src/common/config_validator.cpp

```diff
@@ -37,6 +37,11 @@ void ConfigValidator::validateOptions(cli::mode mode) const {
       validateOptionsParallelData();
       validateOptionsTraining();
       break;
+    case cli::mode::selfadaptive:
+      // validateOptionsTranslation();
+      // validateOptionsParallelData();
+      // validateOptionsTraining();
```

Comment:
Please double check if none of these validations can be used in the self-adaptive mode and remove the comments if that's the case. Try to run at least some of the validations if possible.